### DISCUSSION TITLE
Database SQLite vs MySQL

### DISCUSSION THREAD

**Discussion by Giuseppe-TD** [ZoeyVid/NPMplus#1233] [Q&A] [answered]
What is the point of using a MySQL vs SQLite ? 
If I configure MySQL over SQLite will have a performance improvements or database are only for users sessions?

**Comment by Zoey2936** [comment]
no, even saving it as a json file would make no difference

**Reply by Zoey2936** [reply, marked as answer]
just use sqlite

**Comment by Giuseppe-TD** [comment]
Oh ok thanks
